### PR TITLE
Config as code - links for Git providers access token instructions

### PR DESCRIPTION
**Pull request by shipra-octopus**
Add in links to git providers for users to easily get their personal access token.

### CODE REVIEW

File: docs/projects/version-control/index.md [hunk 1]
```diff
@@ -59,7 +59,12 @@ The _Default Branch Name_ is the branch on which the Octopus configuration will
 
 The default branch must exist.
 
-The _Authentication_ field specifies the credentials used by Octopus when authenticating with the git provider.  The password will likely be a personal access token. 
+The _Authentication_ field specifies the credentials used by Octopus when authenticating with the git provider.  For the Password field, we recommend using a personal access token where possible. Click on the relevant link below to get your personal access token -
```
Comment (harrisonmeister) [inline docs/projects/version-control/index.md:62]:
```suggestion
The _Authentication_ field specifies the credentials used by Octopus when authenticating with the git provider.  For the Password field, we recommend using a personal access token. Git providers allow you to create an access token in different ways:
```

File: docs/projects/version-control/index.md [hunk 2]
```diff
@@ -59,7 +59,12 @@ The _Default Branch Name_ is the branch on which the Octopus configuration will
...
+
+* [GitHub - Creating a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
```
Comment (harrisonmeister) [inline docs/projects/version-control/index.md:64]:
```suggestion
* [GitHub](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
```

File: docs/projects/version-control/index.md [hunk 3]
```diff
@@ -59,7 +59,12 @@ The _Default Branch Name_ is the branch on which the Octopus configuration will
...
+* [Azure DevOps - Use personal access token](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate)
```
Comment (harrisonmeister) [inline docs/projects/version-control/index.md:65]:
```suggestion
* [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate)
```

File: docs/projects/version-control/index.md [hunk 4]
```diff
@@ -59,7 +59,12 @@ The _Default Branch Name_ is the branch on which the Octopus configuration will
...
+* [BitBucket - Personal access tokens](https://confluence.atlassian.com/bitbucketserver063/personal-access-tokens-972354166.html)
```
Comment (harrisonmeister) [inline docs/projects/version-control/index.md:66]:
```suggestion
* [BitBucket](https://confluence.atlassian.com/bitbucketserver063/personal-access-tokens-972354166.html)
```

File: docs/projects/version-control/index.md [hunk 5]
```diff
@@ -59,7 +59,12 @@ The _Default Branch Name_ is the branch on which the Octopus configuration will
...
+* [GitLab - Personal access tokens](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
```
Comment (harrisonmeister) [inline docs/projects/version-control/index.md:67]:
```suggestion
* [GitLab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
```

**Review by harrisonmeister — COMMENTED** [review]
Made some minor suggestions, but happy with it as-is

**Review by harrisonmeister — APPROVED** [review]
LGTM. @shipra-octopus - let me know if you want me to merge or wait for @MJRichardson to also review 😄

**Comment by shipra-octopus** [comment]
Thanks @harrisonmeister. Maybe wait for @MJRichardson . Hopefully he can have a quick look today :-) I have nudged him on slack.

**Review by MJRichardson — APPROVED** [review]
Looks good!

**Pull request merged**